test/sequential/rolldown.forceUpdate.test.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ *
+ * @group sequencerCancellation
+ */
+
+import {
+  ChainName,
+  SequencerStaking,
+} from "../../utils/rollDown/SequencerStaking";
+import {
+  L2Update,
+  Rolldown,
+  createAnUpdate,
+  createAnUpdateAndCancelIt,
+  leaveSequencing,
+} from "../../utils/rollDown/Rolldown";
+import { getApi, initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { BN_MILLION, signTx } from "gasp-sdk";
+import {
+  ExtrinsicResult,
+  findEventData,
+  waitNewBlock,
+  waitSudoOperationSuccess,
+} from "../../utils/eventListeners";
+import { isBadOriginError } from "../../utils/utils";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+import { AssetWallet, User } from "../../utils/User";
+import { Assets } from "../../utils/Assets";
+import { GASP_ASSET_ID } from "../../utils/Constants";
+
+let api: any;
+let testUser: User;
+let testUserAddress: string;
+let chain: any;
+
+async function setupASequencer(user: User, chain: ChainName = "Ethereum") {
+  const extrinsic = await SequencerStaking.provideSequencerStaking(
+    (await SequencerStaking.minimalStakeAmount()).addn(1000),
+    chain,
+  );
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(user),
+    Sudo.sudoAs(user, extrinsic),
+  );
+}
+
+beforeAll(async () => {
+  await initApi();
+  setupUsers();
+  await setupApi();
+  api = getApi();
+  chain = "Ethereum";
+});
+
+beforeEach(async () => {
+  //TODO: Replace this by some monitoring of the active queue.
+  const activeSequencers = await SequencerStaking.activeSequencers();
+  for (const chain in activeSequencers.toHuman()) {
+    for (const seq of activeSequencers.toHuman()[chain] as string[]) {
+      if (seq !== null) {
+        await leaveSequencing(seq);
+      }
+    }
+  }
```

Comment:
same here, I think you already refactored this  some time ago.

Reply:
Added the function